test/rust-test-util/test-result-reporting-client/src/lib.rs

```diff
@@ -9,7 +9,8 @@
     clippy::match_same_arms,
     clippy::must_use_candidate,
     clippy::missing_errors_doc,
-    clippy::missing_panics_doc
+    clippy::missing_panics_doc,
+    clippy::expl_impl_clone_on_copy, // needed for `EnumSetType` derive
```

Comment:
Tried to do a single-line allow but didn't seem to work. The thing I needed to allow was a macro which led to complications.

Reply:
Is this change needed? I was curious what the clippy error was so I checked out your branch, removed this new line, and ran `cargo clippy`, but there were no errors reported

Reply:
Make sure you clean the project before running clippy:
`cargo clean -p trc-client`